### DISCUSSION TITLE
Is there any commercial receiver with GPS telemetry support?

### DISCUSSION THREAD

**Discussion by RobertoD91** [ExpressLRS/ExpressLRS#2705] [Q&A] [answered]
Hi, are there any commercially available PWM ELRS receivers with a USART plug for directly connecting GPS without a flight controller?

**Comment by mha1** [comment, marked as answer]
No, ELRS firmware doesn't support decoding raw GPS data like NMEA. Maybe this helps: https://www.youtube.com/watch?v=HKKmED2W0Uk

**Reply by RobertoD91** [reply]
Ok, so i need to use a flight controller or some board like mateksys CRSF-PWM-V10, thanks!

**Comment by sp3swj** [comment]
Use  RP2040ZERO plus gps module and this works with any Radiomaster PWM receiver, but best use  ERC6 with hardware serial port.

https://github.com/mstrens/oXs_on_RP2040

tested, works 2Hz refresh data


https://photos.app.goo.gl/GAGDPGb8QVfkvvuVA



https://photos.app.goo.gl/yP7qGGJ3w2bR4aWu9

**Reply by tmur11** [reply]
Would a GT-U7 GPS Module work with a Radiomaster PWM reciever?

**Reply by mwlaboratories** [reply]
That is awesome! At what update speed can you get it to run? I'd like to use this input port on the receiver to enter loadcell telemetry to provide forcefeedback to the steering wheel for an fpv car. It would need to be 50hz I'd think for good signal.

**Reply by RobertoD91** [reply]
Also give a look to this one: https://github.com/dgatf/msrc

**Comment by atomGit** [comment]
[ERS-GPS - Precision GPS Sensor for compatible ExpressLRS PWM receivers - RadioMaster RC](https://radiomasterrc.com/products/ers-gps-precision-gps-sensor-for-compatible-expresslrs-pwm-receivers)
